test/index.js

```diff
@@ -214,6 +214,39 @@ test.cb('accepts template object and generates html', (t) => {
   project.compile()
 })
 
+test.cb('generates error if template has an error', (t) => {
+  const locals = {}
+  const rooftop = new Rooftop({
+    name: process.env.name,
+    apiToken: process.env.token,
+    addDataTo: locals,
+    contentTypes: [{
+      name: 'case_studies',
+      template: {
+        path: '../template/error.html',
+        output: (item) => `posts/${item.title}.html`
+      }
+    }]
+  })
+
+  const projectPath = path.join(__dirname, 'fixtures/default')
+  const project = new Spike({
+    root: projectPath,
+    posthtml: { defaults: [exp({ locals })] },
```

Comment:
Yea I think a new PR would be best. I saw #25 is already up, I wasn't sure when that would go in.